.eslintrc.json

```diff
@@ -93,4 +93,4 @@
       }
     }
   ]
-}
+}
```

Comment:
Is it possible to specify the lint rule in the root `.eslintrc.json`, since all the other `.eslintrc.json` extend from it?